premium/usage.go

```diff
@@ -200,6 +206,20 @@ func NewUsageClient(meta plugin.Meta, ops ...UsageClientOptions) (UsageClient, e
 			TeamNameValue: u.teamName,
 		}, nil
 	}
+	// If user wants to use the AWS Marketplace for billing, don't even try to communicate with CQ API
+	if isAWSMarketplace() {
+		cfg, err := awsConfig.LoadDefaultConfig(context.TODO())
```

Comment:
What's the reason for using `context.TODO()` in favor of `context.Background()`? Maybe we should add timeout as well for loading the config?

Also, can we put this logic in a `setupAWSMarketplace` function as `NewUsageClient` is getting quite long

Reply:
`context.TODO()` is supposed to be used when the surrounding function doesn't have a context available to use, so that is why I used it here. I am not sure the value of a timeout would add here.



I have moved